README.md

```diff
@@ -186,3 +186,99 @@ export HIGHLANDS_SSO_PASSWORD='<YOUR_PASSWORD>'
+
+#### Clearstream Success Response
+```
+{
+    "status": 202,
+    "data": {
+        "clearstream_msg": "{\"id\":10,\"sent_to_clearstream\":\"2019-05-15T13:49:14.326Z\",\"sms_json\":\"{\\\"data\\\":{\\\"id\\\":118487,\\\"status\\\":\\\"QUEUED\\\",\\\"sent_at\\\":\\\"2019-05-15T13:49:55+00:00\\\",\\\"completed_at\\\":null,\\\"text\\\":{\\\"full\\\":\\\"Leadership Team: Come in now for 50% off all rolls!\\\",\\\"header\\\":\\\"Leadership Team\\\",\\\"body\\\":\\\"Come in now for 50% off all rolls!\\\"},\\\"lists\\\":[],\\\"subscribers\\\":[\\\"+17707651573\\\"],\\\"stats\\\":{\\\"recipients\\\":1,\\\"failures\\\":0,\\\"throughput\\\":0,\\\"replies\\\":0,\\\"opt_outs\\\":0},\\\"social\\\":{\\\"twitter\\\":{\\\"enabled\\\":false,\\\"id\\\":null,\\\"url\\\":null},\\\"facebook\\\":{\\\"enabled\\\":false,\\\"id\\\":null,\\\"url\\\":null}}}}\",\"got_response_at\":\"2019-05-15T13:49:14.571Z\",\"clearstream_response\":\"QUEUED\",\"created_at\":\"2019-05-15T13:49:14.335Z\",\"updated_at\":\"2019-05-15T13:49:14.571Z\"}"
+    },
+    "error": null
+}
+```
+
+#### Sendgrid Success Response
+```
+{
+    "status": 202,
+    "data": {
+        "sendgrid_msg": "{\"id\":6,\"sent_to_sendgrid\":\"2019-05-15T19:35:09.555Z\",\"mail_json\":{\"from\":{\"email\":\"app1@highlands.org\"},\"subject\":\"Picnic Saturday Week\",\"personalizations\":[{\"to\":[{\"email\":\"bob.p.k.brown@gmail.com\",\"name\":\"Bob Brown\"}],\"dynamic_template_data\":{\"header\":\"Karate Chop\",\"section1\":\"Try to imagine all life as you know it stopping instantaneously and every molecule in your body exploding at the speed of light.\",\"section2\":\"Maybe now you'll never slime a guy with a positron collider, huh?\",\"section3\":\"You will perish in flame, you and all your kind! Gatekeeper!\",\"button\":\"Reply\",\"email_subject\":\"Picnic Saturday Week\"}}],\"template_id\":\"d-f986df533e514f978f4460bedca50db0\"},\"got_response_at\":\"2019-05-15T19:35:09.639Z\",\"sendgrid_response\":\"202\",\"read_by_user_at\":null,\"created_at\":\"2019-05-15T19:35:09.555Z\",\"updated_at\":\"2019-05-15T19:35:09.639Z\"}"
+    },
+    "error": null
+}
+```
+
+ret is created in MessageDirector.create_message(message_vo):
+
+```
+    if errs.size > 0
+      ReturnVo.new({value: nil, error: error_json = return_error(errs, :unprocessable_entity)})
+    else
+      message = Message.create!(msg_target_id: msg_target_id,
+                                manager_id: manager_id, # <= source of message (an application)
+                                receiver_id: receiver.id,
+                                team_id: team.id, # <= message coming from this team
+                                email_subject: message_vo.email_subject,
+                                email_message: message_vo.email_message,
+                                template_id: template.id,
+                                sms_message: message_vo.sms_message)
+
+      ReturnVo.new({value: message, error: nil})
+    end
+```
+
+
+## Message Table
```

Comment:
I don't know how hard this is to be changed now.. But It would be cool if we have a enum or a string representation for 'sendgrid' and 'clearstream' rather than numbers (1,2..3). It is more readable, and at a first glance the person can understand it. 

If we have  strings we could do a validation to make sure the strings would be only `clearstream` or `sendgrid`.

Reply:
Oh, you're talking about in **message_director.rb**, right?

Example:
```
    case ret.value.msg_target.name
    when 'Sendgrid'

```

In this case, we need the literal string 'Sengrid' or 'Clearstream' a few lines below that.  An enum won't help us in this case.

Not sure if it's even a good idea to create write extra code to create a constant for this.  This is the only place those strings MsgTarge.name values are checked.